### PR TITLE
fix: broken top up e2e tests

### PR DESCRIPTION
**Pull request by pandey-shubh**
## Pull request type

Please check the type of change your PR introduces:

-  Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

- broken top up test

Issue Number: N/A

## What is the new behavior?

- fixed broken top up tests

### CODE REVIEW

**Review by ArpitAgrawal321 — APPROVED** [review]
lgtm

File: tests/e2e/topup/topup.test.ts [hunk 1]
```diff
@@ -2,48 +2,57 @@
...
+  await page.click("img[alt=moonpay]");
   await page.fill("input[type='number']", "100");
   const usdToSol100 = Number(await getInnerText(page, "#resCryptoAmt"));
   expect(usdToSol100).toBeGreaterThan(0);
 
-  // set amount to be transfered as 200 US Dollars, expect a positive value for expected SOL
+  // set amount to be transferred as 200 US Dollars, expect a positive value for expected SOL
   await page.fill("input[type='number']", "200");
   const usdToSol200 = Number(await getInnerText(page, "#resCryptoAmt"));
   expect(usdToSol200).toBeGreaterThan(0);
```
Comment (ishan-torus) [inline tests/e2e/topup/topup.test.ts:24]: also add a case that usdToSol200 should be > usdToSol100

File: tests/e2e/topup/topup.test.ts [hunk 2]
```diff
@@ -2,48 +2,57 @@
...
-  // ensure that on clicking save, it is refirected to Ramp Payment page having torus logo
-  await Promise.all([page.click("button:has-text('Top up')"), page.waitForEvent("popup")]);
-  await expect((await page.$$("img[alt=Ramp]")).length).toEqual(1);
+  // ensure that on clicking Top up, it is redirected to MoonPay Payment page
+  const [page2] = await Promise.all([page.waitForEvent("popup"), page.click("button:has-text('Top up')")]);
+  // closing moonpay page, will show an error on top up page
+  page2.close();
+  await wait(500);
+  await ensureTextualElementExists(page, "Transaction could not complete.");
 });
```
Comment (ishan-torus) [inline tests/e2e/topup/topup.test.ts:33]: Add test for changing of `fiat` currency or `crypto` currency also changes the value as expected.

**Comment by ishan-torus** [comment]
lgtm @metallicalfa2

**Pull request merged**